R/zoop_downloader.R

```diff
@@ -704,6 +851,7 @@ Zoopdownloader <- function(
       dplyr::group_by(dplyr::across(-.data$CPUE))%>%
       dplyr::summarise(CPUE=sum(.data$CPUE, na.rm=TRUE))%>% #Some taxa now have the same names (e.g., CYCJUV and OTHCYCJUV) so we now add those categories together.
       dplyr::ungroup()%>%
+      dplyr::left_join(stations, by=c("Source", "Station"))%>%
```

Comment:
As before, can you move this 1 line down to be the last line of this pipe?

Reply:
Yup!